### ISSUE TITLE
Split recipe in multiple outputs

### ISSUE THREAD

**Issue by beenje** [conda-forge/omniorb-feedstock#15]
Hello!
We use omniorb to build [cpptango](https://github.com/conda-forge/cpptango-feedstock).
I realised that omniorb has python as run dependency, which forces us to install python when installing cppTango.
As it's a C++ library, this dependency shouldn't be needed.
I wonder if we could not create multiple outputs:
- one with the libs but without python as dependency: omniorb-c++? or another name
- maybe omniorb-py for the python only dependency (or does it rely on the c++ libs)?
- and keep omniorb for everything

Do you think it would make sense?

**Comment by carlodri** [comment]
Hi @beenje thanks for the report!

Actually I have kind of stepped far away from omniorb and I don't use it anymore. My original plan at the time was to build pytango here con conda-forge, but then I dropped the idea due to other priorities.

I don't know if I will have the time to tackle this issue, for which your solution makes perfect sense to me AFAIU. If you have the time, you can open a PR here with the necessary recipe changes (for instance along the lines of https://github.com/carlodri/black-feedstock/blob/master/recipe/meta.yaml), and I will merge it as soon as it is ready. If you want, you can also be added as a maintainer to the feedstock.

**Comment by beenje** [comment]
Thanks @carlodri, I'll give it a try when I find some time!

For being added as a maintainer, it could make sense as I'm involved with tango conda packages, even if I don't want to touch omniorb too much :-) Let's wait for my PR.

**Comment by carlodri** [comment]
as you have seen I have already tried #16 but it seems like to run `omniidl` Python is needed on the target systems....does this make sense to you? I'm really too ignorant of `omniorb`.... 😬

**Comment by beenje** [comment]
Yes, that's not gonna work. Indeed `omniidl` is a Python script and there are also some Python modules installed so we can't just remove Python dependency.
But it should be possible to create an "omniorb-libs" output with only the libs (and no Python dependencies).
